packages/@okta/vuepress-site/docs/reference/api/authorization-servers/index.md

```diff
@@ -678,13 +678,13 @@ curl -v -X GET \
 
 ##### Response example
 
-Returns a list of [Policy Rule](#rule-object) defined in the specified Custom Authorization Server and Policy
+Returns a list of [Policy Rules](#rule-object) defined in the specified Custom Authorization Server and Policy
```

Comment:
Returns a list of [Policy Rules](#rule-object) **that are** defined in the specified Custom Authorization Server and Policy